lib/index.js

```diff
@@ -179,10 +182,10 @@ module.exports = class OtpRunner {
       errors.push('At least one of `buildGraph` or `runServer` must be set to true')
     }
 
-    // if build is set to true, then gtfsAndOsmUrls needs to be defined
-    if (buildGraph && (!gtfsAndOsmUrls || !gtfsAndOsmUrls.length)) {
-      errors.push('`gtfsAndOsmUrls` must be populated for graph build')
-      console.log(gtfsAndOsmUrls)
+    // if build is set to true, then routerFolderDownloads needs to be defined
+    if (buildGraph && (!routerFolderDownloads || !routerFolderDownloads.length)) {
```

Comment:
Would a routerFolderDownloads = 'some-string-value' be caught in validation before this line?

Reply:
Yes, there is a JSON Schema validation that occurs before this.